HiP-EventStoreLib/ICustomEvent.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace PaderbornUniversity.SILab.Hip.EventSourcing
+{
+    /// <summary>
+    /// Interface for events that have additional information (e.g. in metadata) and need to be serialized/deserialized diffently
+    /// </summary>
+    public interface ICustomEvent : IEvent
```

Comment:
I'm not that happy with the name "custom event"... Isn't, for example, DataStore's `ExhibitCreated`-event even more "custom" (in terms of "tailored to a specific problem domain") than the new, quite generic `PropertyChangedEvent`?

Naming things isn't easy... maybe `IEventWithMetadata`? Hm... 🤔